workflow-bot-app/src/main/java/com/symphony/bdk/workflow/api/v1/WorkflowsMgtApi.java

```diff
@@ -55,4 +59,13 @@ ResponseEntity<Void> deleteSwadlById(
       @ApiParam(value = "Workflow's token to authenticate the request", required = true)
       @RequestHeader(name = X_MANAGEMENT_TOKEN_KEY) String token,
       @ApiParam(value = "Workflow's id that is provided in SWADL", required = true) @PathVariable String id);
+
+  @ApiOperation("Streaming logs in SSE.")
+  @ApiResponses(value = {@ApiResponse(code = 200, message = "", response = ResponseBodyEmitter.class),
+      @ApiResponse(code = 401, message = "Request is not authorised", response = ErrorResponse.class)})
+  @ResponseStatus(HttpStatus.OK)
+  @GetMapping(path = "/logs", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
+  SseEmitter streamingLogs(
```

Comment:
Isn't it missing the token request header?

Reply:
added